app/models/course.rb

```diff
@@ -14,6 +14,12 @@ class Course < ApplicationRecord
   validates :status, presence: true, inclusion: {in: statuses.keys}
   validates :day_start, presence: true
   validate  :day_start_is_lower_than_day_end
+  default_scope {order('created_at DESC')}
+  scope :search_by_name, ->(name){where "lower(name) LIKE ?", "%#{name}%"}
+  scope :get_courses, ->(course_ids){where id: course_ids}
```

Comment:
Đang ở model Course rồi. Em đặt tên ngắn gọn như này thôi
```suggestion
  scope :by_ids, ->(ids){where id: ids}
```